### PR TITLE
add Loading Component with animation

### PR DESCRIPTION
**Pull request by Juett**
add Loading Component with animation

### CODE REVIEW

File: js/components/ChooseAvatar/styles.js [hunk 1]
```diff
@@ -51,20 +56,24 @@ const styles = StyleSheet.create({
     height: height / 9
   },
   editAvatar: {
+    marginLeft: 20,
+    marginRight: 20,
+    marginTop: 15,
+    marginBottom: 15,
```
Comment (siddharthparmar7) [inline js/components/ChooseAvatar/styles.js]: can you work without this huge margin?

File: js/screens/Journal/Journal.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Text, View } from "react-native";
+import Loading from "../../components/Loading";
 
 const Journal = () => {
   return (
     <View>
```
Comment (siddharthparmar7) [inline js/screens/Journal/Journal.js]: use Fragment instead.

File: js/components/Loading/Loading.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+import {
+  Text,
+  View,
+  Image,
+  ImageBackground,
+  Easing,
+  Animated,
+  Dimensions,
+  StyleSheet
+} from "react-native";
```
Comment (siddharthparmar7) [inline js/components/Loading/Loading.js:9]: please remove unnecessary imports.

File: js/components/Loading/styles.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import { StyleSheet, Dimensions } from "react-native";
+import globalStyles from "../../config/styles";
```
Comment (siddharthparmar7) [inline js/components/Loading/styles.js]: remove unnecessary imports.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Don't forget to squash your commits. Also, your feature branch should follow the conventions. Please make the requested changes. Thank you.

File: js/components/ChooseAvatar/styles.js (hunk 1 above)
Comment (Juett) [reply js/components/ChooseAvatar/styles.js]: I tried everything I could not to use these huge margins but its the only way to match the design.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
your branch name is not correct. Use the conventions we showed you.

**Pull request closed without merging**